<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>开始监控</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>停止监控</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total">
        <property name="text">
         <string>总文件数：0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_success">
        <property name="text">
         <string>发送成功：0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_failed">
        <property name="text">
         <string>发送失败：0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_Log"/>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_currentFile">
      <property name="text">
       <string>当前发送的文件：无</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_speed">
      <property name="text">
       <string>传输速度：0M/s</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTCP_Client">
    <property name="title">
     <string>TCP Client</string>
    </property>
   </widget>
   <addaction name="menuTCP_Client"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
